Amazon.QLDB.Driver.IntegrationTests/StatementExecutionTests.cs

```diff
@@ -626,7 +624,7 @@ public void Execute_UpdateSameRecordAtSameTime_ThrowsOccException()
         }
 
         [TestMethod]
-        [DynamicData(nameof(CreateIonValues), DynamicDataSourceType.Method)]
+        [CreateIonValues]
```

Comment:
Does this run correctly?

Reply:
Yes, it runs and passes.  DynamicData, of what I gather, can only be used within the compilation unit.  In order to use data outside the test, we must use ITestDataSource interface: https://dev.to/frannsoft/mstest-v2---new-old-kid-on-the-block